java/src/main/java/actorworkflow/EmailStrings.java

```diff
@@ -0,0 +1,12 @@
+package actorworkflow;
+
+public interface EmailStrings {
```

Comment:
im not sure that using interface for this is best practice? can be just static final fiends in some util class if needed.

Reply:
Been a [long] while since I've written Java, and I saw this pattern used somewhere and so ran with it. Reading up on the "Constant Interface Anti-pattern", I'm now moving these to classes.